src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -158,6 +160,35 @@ export function annotateMutationPutativeDriver(
     ) as AnnotatedMutation;
 }
 
+export function annotateStructuralVariantPutativeDriver(
+    structuralVariant: StructuralVariant,
+    putativeDriverInfo: {
+        oncoKb: string;
+        hotspots: boolean;
+        cbioportalCount: boolean;
+        cosmicCount: boolean;
+        customDriverBinary: boolean;
+        customDriverTier?: string;
+    }
+): AnnotatedStructuralVariant {
+    const putativeDriver = !!(
+        putativeDriverInfo.oncoKb ||
+        putativeDriverInfo.hotspots ||
+        putativeDriverInfo.cbioportalCount ||
+        putativeDriverInfo.cosmicCount ||
+        putativeDriverInfo.customDriverBinary ||
+        putativeDriverInfo.customDriverTier
+    );
+    return Object.assign(
```

Comment:
curious why you need Object.assign?  why not just add structuralVariant in the same way as putativeDriver?

Reply:
We could. I was copy pasting logic from `annotateMutationPutativeDriver` (function above this)